### PR TITLE
Codeowners revision

### PR DESCRIPTION
**Pull request by jinchung**
Revisions to CODEOWNERs to aim for a more minimal set of critical files and directories that need to remain with CODEOWNERs.

### CODE REVIEW

**Review by brunobar79 — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -1,217 +1,80 @@
...
 
 # SECURITY / APP INFRA
-yarn.lock @brunobar79 @jinchung
-
-# Performance Tracking
-src/performance-tracking @terrysahaidak @jkadamczyk
-android/app/src/main/java/me/rainbow/NativeModules/RNStartTime @terrysahaidak @jkadamczyk
-ios/RNStartTime.h @terrysahaidak @jkadamczyk
-ios/RNStartTime.m @terrysahaidak @jkadamczyk
+yarn.lock @brunobar79 @estebanmino @nickbytes
```
Comment (osdnk) [inline CODEOWNERS:80]: can we add more people here? I.e., someone not from America?

File: CODEOWNERS [hunk 2]
```diff
@@ -1,217 +1,80 @@
 # Rainbow CODEOWNERS file
 
 # ANDROID
-android/ @brunobar79 @osdnk @terrysahaidak
-
-# E2E
-config/ @brunobar79 @skylarbarrera @osdnk
-e2e/ @brunobar79 @ibrahimtaveras00 @skylarbarrera
+android/ @brunobar79 @osdnk @terrysahaidak @estebanmino
+android/app/src/main/java/me/rainbow/NativeModules/RNStartTime @terrysahaidak @jkadamczyk
```
Comment (osdnk) [inline CODEOWNERS:5]: is this module so crucial?

File: CODEOWNERS [hunk 3]
```diff
@@ -1,217 +1,80 @@
...
-src/hooks/useENSRegistration.ts @estebanmino @jxom
-src/hooks/useENSRegistrationCosts.ts @estebanmino @jxom
-src/hooks/useGas.ts @estebanmino @brunobar79 @jinchung
-
-# Languages
-src/languages @skylarbarrera @marco
-
+src/handlers/authentication.ts @brunobar79 @estebanmino
+src/handlers/cloudBackup.ts @brunobar79 @skylarbarrera @estebanmino
+src/handlers/fedora.ts @osdnk @estebanmino
```
Comment (osdnk) [inline CODEOWNERS:23]: Terry?

File: CODEOWNERS [hunk 4]
```diff
@@ -1,217 +1,74 @@
...
 
 # ANDROID
-android/ @brunobar79 @osdnk @terrysahaidak
-
-# E2E
-config/ @brunobar79 @skylarbarrera @osdnk
-e2e/ @brunobar79 @ibrahimtaveras00 @skylarbarrera
-
-# IOS
-ios/ @brunobar79 @osdnk
```
Comment (terrysahaidak) [inline CODEOWNERS:11]: I would also keep the ios folder since we have android one

**Review by terrysahaidak — APPROVED** [review]
Looks good to me overall

**Review by estebanmino — APPROVED** [review]

**Review by skylarbarrera — APPROVED** [review]

**Pull request merged**